Update cdnjs (#1713)
github/explore: topics/cdnjs/index.md

---

> created_by: Matt Cowley, Sven Sauleau, Tyler Caslin, Drew Freyling, Alex, Ryan Kirkman, Thomas Davis

Reviewer: the website states that Thomas Davis and Ryan Kirkman are the two founders. Perhaps the current version is incorrect as well. Thoughts on using this?
```suggestion
created_by: Ryan Kirkman and Thomas Davis
```

Reviewer 2: 👋🏼 Hi from cdnjs. Ryan & Thomas are indeed our founders, but they’re now rather inactive. The team listed on the website and here are the ones that now maintain and continue to create cdnjs. I guess this is a question of wether this field is for original creators (founded in 2011), or those who currently run it?

Reviewer: The intention behind `created_by` is to define the original creators, see github.com/topics/js for a good example of that. I think we mistakenly included a wide net when this topic was first created. I'm happy to make those changes outside of this PR, or you can include them here.

display_name: cdnjs

> github_url: https://github.com/cdnjs/packages

Reviewer: What is the intention of this change? cdnjs/cdnjs still seems like a valid repo to point to.

Reviewer 2: I can probably provide some context here: cdnjs/cdnjs is the repo that contains the content hosted on the CDN, but is now locked down and robot-only. cdnjs/packages is the new repo that we encourage folks to contribute to, where all the hosted packages are defined.

I assume the intention with this change is to keep this file pointing at a contributor-friendly repo?

Reviewer: I just checked and we support org level links here. This would be more appropriate in this instance since we aren't using this page to direct visitors on how to contribute to cdnjs, it's more of a place to give context on what a topic is.
```suggestion
github_url: https://github.com/cdnjs
```

url: https://cdnjs.com/
logo: cdnjs.png
related: cdn, javascript, css, html, library, package, opensource, foss
topic: cdnjs

> short_description: The \#1 free and open source CDN built to make life easier for developers.

Reviewer: Thoughts on moving this to something more objective?
```suggestion
short_description: A free and open source CDN built to make life easier for developers.
```

---

> cdnjs is a free and open source CDN service trusted by over 10% of websites, powered by Cloudflare. Making it faster and easier to load library files on your websites.

Reviewer: Since the number is likely to change, it’d be great if we could omit it to reduce churn. What do you think about this:

```suggestion
A free and open source CDN service powered by Cloudflare. Making it faster and easier to load library files on your websites.
```
